discovery-infra/update_assisted_service_cm.py

```diff
@@ -33,8 +34,26 @@
     ("IMAGE_BUILDER", ""),
     ("OCM_BASE_URL", ""),
     ("PUBLIC_CONTAINER_REGISTRIES", ""),
-    ("CHECK_CLUSTER_VERSION", "")
+    ("CHECK_CLUSTER_VERSION", ""),
+    ("HW_VALIDATOR_REQUIREMENTS", "")
 ]
+MASTER_REQUIREMENTS = {
+    "cpu_cores": 4,
+    "ram_mib": 8192,
+    "disk_size_gb": 10,
+    "installation_disk_speed_threshold_ms": 10
+}
+WORKER_REQUIREMENTS = {
+    "cpu_cores": 2,
+    "ram_mib": 3072,
```

Comment:
From the service code it looks like we require 4GB of RAM, no?

Reply:
According to this line https://github.com/openshift/assisted-test-infra/blob/1ec704a3516cb67b8abfcae92ee498deda409e11/discovery-infra/update_assisted_service_cm.py#L25 it should be 3 GiB here.